### ISSUE TITLE
Allow changes to container timezone to be persistant, or document method to make timezone changes

### ISSUE THREAD

**Issue by FirbyKirby** [mtoensing/Docker-Minecraft-PaperMC-Server#22]
This may be more of a request for support or a feature request than an issue.

I'm running 1.16.3 version of the docker image successfully on a Synology DS218+ (with RAM upgrade.) However, the log files are timestamped using UTC. When examining logs at a later date to determine what the cause of an error or warning was, it's inconvenient to convert these UTC timestamps into a local time zone.

I've attempted to set the local time zone myself by editing the /etc/localtime file in the container, but of course, this change will not be persistent across reboot without mounting the file properly, which I was unable to do.

I believe there are a few other methods, but I am a novice with docker containers, and even more so using the Synology Diskstation Docker package. If the docker image could be modified to allow persistent timezone changes or instructions could be given in the documentation for how to make a timezone change persistent, this would be a large usability improvement.

**Comment by mtoensing** [comment]
Since the container is based on openjdk this should work: https://github.com/docker-library/openjdk/issues/134

**Comment by mtoensing** [comment]
Documented in Readme and tested. Works.

**Comment by FirbyKirby** [comment]
Thanks! Implemented it just now and does just what I wanted.

My only other suggestion would be to add a link to a location for users who are unfamiliar with the timezone environment variable to learn what the valid values are. This isn't the best link but it's probably a good start. Ideally, it would be good for people to know that the most popular method is to reference files in the `zoneinfo` directory of the container (as you did in your example.) At least, this is the path I followed to find the right value for my container.

https://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html

And maybe a link to the tz database?
https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Great work on this image! Keep up the excellent work.
